src/plugins/discord/sqliteMirror.test.js

```diff
@@ -222,11 +223,40 @@ describe("plugins/discord/sqliteMirror", () => {
         channel_id: "2",
         author_id: "3",
         non_user_author: 0,
-        timestamp_ms: 1583278510615,
+        timestamp_ms: 2,
         content: "Just going to drop this here",
       });
     });
 
+    describe("nthMessageIdFromTail", () => {
+      const channelId: Snowflake = "1";
+      const authorId: Snowflake = "2";
+
+      it("retrieves nth message from tail", () => {
+        const db = new Database(":memory:");
+        const sqliteMirror = new SqliteMirror(db, "0");
+        buildNMessages(5, sqliteMirror, channelId, authorId);
+        expect(sqliteMirror.messages(channelId).map((x) => x.id)).toEqual([
```

Comment:
Added this sanity check bc it's cheap and makes it easier to reason about the test